pkg/controllers/user/helm/common.go

```diff
@@ -83,6 +84,8 @@ func helmDelete(tempDirs *common.HelmPath, app *v3.App) error {
 
 func (l *Lifecycle) generateTemplates(obj *v3.App) (string, string, *common.HelmPath, error) {
 	var appSubDir string
+	var commands []string
+	var cmd *exec.Cmd
```

Comment:
Move these down to above the if block, instantiating where you need them helps when scanning the code